DevelopedCalculator.js

```diff
@@ -27,12 +27,16 @@ class DevelopedCalculator extends Calculator {
     gettingHistoryArray() {
         // updating the history element by the history which is joined with ,
         this.historyElement.textContent = this.history.join(",")
+            //  this.historyElement.textContent = this.history.join("")
     };
     onOperatorButtonClick(event) {
+
```

Comment:
stop pushing random changes in every commit 

only change the line that need to be changed